### PR TITLE
chore: ruff cleanup

### PR DESCRIPTION
**Pull request by RazCrimson**
Fixes all formatting by ruff

@nicolargo 
Should we add something like precommit tooling that automatically runs ruff on each and every commit?

Something like would be nice: https://github.com/pre-commit/pre-commit

### CODE REVIEW

**Comment by nicolargo** [comment]
Why not, there is some examples here: https://github.com/astral-sh/ruff?tab=readme-ov-file#installation

We just have to take into account the following thinks in this PR:
- remove all others format targets but ruff in the Makefile (format should run ruff): https://github.com/nicolargo/glances/blob/develop/Makefile#L88-L94
- hava a look on the existing Ruff configuration file: https://github.com/nicolargo/glances/blob/develop/pyproject.toml#L6-L51
-

File: glances/client.py [hunk 1]
```diff
@@ -48,10 +47,10 @@
 
         # Build the URI
         if args.password != "":
-            self.uri = 'http://{}:{}@{}:{}'.format(args.username, args.password, args.client, args.port)
+            self.uri = f'http://{args.username}:{args.password}@{args.client}:{args.port}'
         else:
-            self.uri = 'http://{}:{}'.format(args.client, args.port)
-        logger.debug("Try to connect to {}".format(self.uri))
+            self.uri = f'http://{args.client}:{args.port}'
+        logger.debug(f"Try to connect to {self.uri}")
```
[the remark this answers is not in the document]
Comment (RazCrimson) [reply glances/client.py:53]: @nicolargo Should we avoid logging this?

**Comment by RazCrimson** [comment]
@nicolargo 
Could you give this a spin on your end?

I've just enabled some of the easy to fix lint rules that should not change any behavior.
Enabling all of them would require a lot more changes than this.

I think it would be better to enable the others slowly over other releases as they would require more invasive code changes.

**Comment by nicolargo** [comment]
Review ok on my side, i have to reconfigure my VSCode to set Ruff as the default Linter (instead of Flake8). I just push some minors changes.

PR can be pushed on develop.

**Pull request merged**